apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart

```diff
@@ -214,7 +214,7 @@ class MemberCard extends StatelessWidget {
                 children: [
                   isNotEligible ||
                           isBeneficiaryIneligible ||
-                          isBeneficiaryRefused ||
+                          false ||
```

Comment:
@amitverma-egov This can be removed altogether

Reply:
@vishal-egov sure